decidim-admin/config/locales/en.yml

```
@@ -865,8 +865,8 @@ en:
           remove: Remove
           up: Up
         form:
-          add: Add to whitelist
-          title: External domain whitelist
+          add: Add to allowlist
+          title: External domain allowlist
```

Maybe change it to : 
```suggestion
          title: Allowed list of External domains
```